tests/CMakeLists.txt

```diff
@@ -68,6 +68,17 @@ if(HAVE_PYSCF)
     endif()
   endif()
 endif()
+
+if(HAVE_GPAW)
+    include("${qmcpack_SOURCE_DIR}/CMake/python.cmake")
```

Comment:
If the converter doesn't invoke anything from GPAW. All the HAVE_GPAW should be removed.
Please move the python checks into `gpaw/CMakeLists.txt`

Reply:
This is still open. The converter needs GPAW, and the tests shouldn't be added without it. I've moved the clause down to `tests/converter`, where it controls whether or not to add subdirectory `gpaw`. Also since we're calling the converter in the original directory, there is no need to require `Eshdf` and `Eshdf_gpaw` classes. Is this okay?